### ISSUE TITLE
add grid overlays to gaiagps.com/map

### ISSUE THREAD

**Issue by andrewljohnson** [trailbehind/contract-offers#8]
## Summary

In the mobile version of Gaia GPS, users can overlay multiple kinds of grid on the map:
- Lat/lon
- UTM
- MGRS
- Distance
## Deliverables

The contractor for this job will implement grids on gaiagps.com/map, in close to the same way it works in the iOS app. 
- [View a video of the iOS grids](https://www.dropbox.com/s/obkwnfentxg7a4a/grid-walkthrough.mov) to see the four grid types, and menu control UI.
- [Similar to this other video](https://www.dropbox.com/s/rc2d0x1zo1kzgqh/grid-coordinates.mov), the grids labels should support different label formats for decimal and degrees/minutes.

To work on this project, [fork this repo](https://github.com/trailbehind/leaflet-grids), and submit your eventual code as a pull request.

In summary:
- make it similar to iOS, and suggest improvements
- add grid control to the Overlays menu on gaiagps.com/map
- show same grid overlays choices as in app
- make sure grids are integrated with the printing feature on gaiagps.com/map
## Technical Notes
- You may wish to incorporate code rom an existing leaflet grid plugin, but maybe not.
- It might make sense to separate the UTM or other grids into separate modules - the grids vary in functionality.
## Time Estimate

This should take 1-5 days to implement.
## Qualifications
- Background with leaflet.js is a big plus. 
- Knowledge of GIS and maps probably needed so you know how the grids work overall.

**Comment by drwelby** [comment]
Are there grids in the Android app?

**Comment by andrewljohnson** [comment]
@drwelby No, there are not. Why do you ask?

You can email support@gaiagps.com if you want a copy of the Android app to look at, let them know we're talking about contracts.

**Comment by drwelby** [comment]
> The contractor for this job will implement grids on gaiagps.com/map, in close to the same way it works in the **iOS/Android app.**

I couldn't find them in the Android app.

**Comment by andrewljohnson** [comment]
@drwelby My mistake, I fixed the post now. 

If you don't object, I'll delete our conversation to avoid confusion.

**Comment by drwelby** [comment]
Yes, no need for the comments at this point.
